chameleon/settings.py

```diff
@@ -244,6 +247,33 @@
 
 LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"),)
 
+
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework.authentication.SessionAuthentication"
+    ],
+    # if not set explicitly, make admin only
```

Comment:
👍🏻 -- how do authentication credentials work? Is it token-based? "SessionAuthentication" seems to me that it's working based on the Django cookie.

Reply:
> 👍🏻 -- how do authentication credentials work? Is it token-based? "SessionAuthentication" seems to me that it's working based on the Django cookie.

Docs: https://www.django-rest-framework.org/api-guide/authentication/#sessionauthentication

It's directly using django's session authentication backend, but I'm not familiar with the details. It's worth digging into.